modules/flatgeobuf/src/lib/types.ts

```diff
@@ -0,0 +1,18 @@
+import type {LoaderOptions} from '@loaders.gl/loader-utils';
+
+export type FlatGeobufSupportedShapes =
+  | 'geojson-row-table'
+  | 'columnar-table'
+  | 'geojson'
+  | 'binary';
+
+export type FlatGeobufLoaderOptions = LoaderOptions & {
+  flatgeobuf?: {
+    shape?: FlatGeobufSupportedShapes;
+  };
+  gis?: {
+    reproject?: boolean;
+    _targetCrs?: string;
+    format?: FlatGeobufSupportedShapes;
```

Comment:
Nit: Should we deprecated gis.format? If so, no reason to add more supported types here.
```suggestion
    /** @deprecated Use options.flatgeobuf.shape */
    format?: 'geojson' | 'binary';
```